tests/src/Network/ProbeTest.php

```diff
@@ -117,12 +117,45 @@ public function testGetFeedLinkBase()
 		}
 	}
 
-	public function dataUri()
+	public function dataCleanUri(): array
 	{
 		return [
 			'@-first' => [
-				'uri'         => '@Artists4Future_Muenchen@climatejustice.global',
-				'assertUri'   => 'Artists4Future_Muenchen@climatejustice.global',
+				'expected' => 'Artists4Future_Muenchen@climatejustice.global',
+				'uri'      => '@Artists4Future_Muenchen@climatejustice.global',
+			],
+			'no-scheme-no-fragment' => [
+				'expected' => 'example.com/path?arg=value',
+				'uri'      => 'example.com/path?arg=value',
+			],
+			'no-scheme-fragment' => [
+				'expected' => 'example.com/path?arg=value#fragment',
```

Comment:
Are we really expecting this fragment there?

Reply:
I don't know, but this is how the original function was implemented and I just wanted to ensure we weren't changing its behavior with the switch to the `Uri` object.

Reply:
Understandable. Possibly you could add a comment there, that this is the current return result but it is to discuss if this behaviour is correct?

Reply:
Done.